packages/styles/scss/themes/expressive/components/_ui-shell.scss

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/layout/convert';
+
+//-----------------------------
+// Accordion (expressive)
+//-----------------------------
+
+/// Accordion styles (expressive)
+/// @access private
+/// @group accordion-expressive
+@mixin ui-shell-expressive {
+  $side-nav-link-padding: calc(
+    (#{$container-02} - #{map-get($body-long-01, 'line-height')}) / 2
+  );
+  $switcher-nav-link-padding: calc(
+    (#{$container-02} - #{map-get($productive-heading-01, 'line-height')}) / 2
+  );
```

Comment:
Seems that side nav item doesn't word wrap (I saw ellipsis instead). In such case, we can simply use flex box, etc. to make the content centered regardless of the container size. `padding-top: calc(...)` approach is an escape hatch for the cases where the height is variable (and thus centering does not yield to the right padding).

Reply:
yeah I initially had the padding set, but saw carbon sets the `height` so I had to override it. I can get rid of the `padding` since the height is set.